virttest/qemu_devices/qdevices.py

```diff
@@ -1910,6 +1911,12 @@ def start_daemon(self):
         self.set_param('run_bg_kwargs', {'output_func': self._handle_log,
                                          'auto_close': False})
         super(QVirtioFSDev, self).start_daemon()
+        if not self.is_daemon_alive():
+            output = self.daemon_process().get_output()
+            self.daemon_process().close()
```

Comment:
Avoid leaking, we should reset the `self._daemon_process = None`.
So I suggest adding a method to do this in the base class:

```
def close_daemon_process(self):
    self._daemon_process.close()
    self._daemon_process = None    
```

```suggestion
            self.close_daemon_process()
```

Reply:
Thanks a lot, corrected.